docs/guides/deployment-patterns.md

```diff
@@ -0,0 +1,170 @@
+
+TODO: Test flow name
+
+```python
+from my_project.flows import calculate_average_commits
+from prefect.deployments import Deployment
+
+@flow
+def build_deployments(name):
+    deployment = Deployment.build_from_flow(
+    flow=name,
+    name=f"{name}-deployment", 
+    version=1, 
+    work_queue_name="default",
+    work_pool_name="default-agent-pool",
+)
+    deployment.apply()
+
+if __name__ == "__main__":
+    build_deployments(calculate_average_commits)
+```
+
+### Event Triggered Flow
+
+![Event diagram](img/guides/event.png)
+
+With Prefect’s automations release, a new pattern is now possible. Whenever a flow run changes state, as it does when it starts or completes running, it emits an event signaling the change to the rest of the system. With automations, Prefect can trigger flows based on a specific flow run state change, or lack thereof. Just as with a submitted flow, a triggered flow can run on separate infrastructure from the initial flow.
+
+This pattern is most useful when you want conceptual, execution, and awareness separation. In this pattern, the triggered flow doesn’t need to know anything about the initial flow. The event that triggers it could come from anywhere. It could even be one of several events that the trigger is listening for. We expect that users will adopt this pattern for the similar reasons that any organization might adopt an event-driven architecture. The loose coupling enables more distributed software and minimizes coordination costs.
+
+For example, you could send an event in your workflow where an Automation can kick off another deployment based off its response. 
+
+```python
+def some_function(name: str="Marvin") -> None:
+    print(f"hi {name}!")
+    emit_event(event=f"{name}.sent.event!", resource={"prefect.resource.id": f"test-event-{name}"})
+
+```
+You are able to emit an event within a function to have more lightweight interaction with Prefect. The only necessary fields are the event name and the resource.id which you are able to trigger different actions. Create custom triggers from events like down below:
+```json
+{
+  "match": {
+    "prefect.resource.id": "test-event-Marvin"
+  },
+  "match_related": {},
+  "after": [],
+  "expect": [
+    "Marvin.sent.event!"
+  ],
+  "for_each": [],
+  "posture": "Reactive",
+  "threshold": 1,
+  "within": 0
+}
+``` 
+
+With events you are able to explore different responses, and have a fine tune control over the full functionality of [Automations](https://docs.prefect.io/2.10.21/cloud/automations/).
+
+Keep in mind, this event driven workflow allows you to supercharge interactions between multiple workspaces by exploring event webhooks. Find more information on how to do so in the [event webhooks](https://docs.prefect.io/2.10.21/cloud/webhooks/#webhook-templates) documentation. 
+
```

Comment:
i would have though this section would more explicitly show an example of how to run an event driven flow from a deployment, considering the focus of this page is deployment patterns. emitting the event seems like only part of the story

Reply:
I plan on creating a separate event driven guide on showing different usescases we can do with this event driven pattern, if it seems like a better user experience adding that content in here, I can do that!

Reply:
My plan was to hyperlink that additional documentation at the tail end of this paragraph. The main focus was to provide a taste on what else they can do with this simple code example with events + automations, and then provide a link to learn more about this specific pattern.